validphys2/src/validphys/photon_pdf/compute_photon.py

```diff
@@ -0,0 +1,153 @@
+"""Script that calls fiatlux to add the photon PDF."""
+
+# from validphys import lhapdfset
+from validphys.api import API
+import lhapdf
+import fiatlux
+import numpy as np
+from eko.couplings import Couplings
+from eko.compatibility import update_theory
+from eko.runner import Runner
+from .structure_functions import StructureFunction
+
+import yaml
+
+class Photon:
+    def __init__(self, theoryid, fiatlux_runcard, replica=0):
+        # TODO : for the moment we do the 0-th replica then we change it
+        self.theory = API.theoryid(theoryid = theoryid.id).get_description().copy()
```

Comment:
If theoryid is passed to performfit though vp this is already being done. i.e. you can do direclty 

`self.theory = theoryid.get_description()`